src/core/include/units/quantity.h

```diff
@@ -133,13 +198,15 @@ class quantity {
   }
 
   // construction, assignment, destruction
-  quantity() = default;
-  quantity(const quantity&) = default;
-  quantity(quantity&&) = default;
+  quantity() noexcept(std::is_nothrow_default_constructible_v<rep>) = default;
+  quantity(const quantity&) noexcept(std::is_nothrow_copy_constructible_v<rep>) = default;
+  quantity(quantity&&) noexcept(std::is_nothrow_move_constructible_v<rep>) = default;
```

Comment:
As I wrote already this is not needed. `=default` will add `noexcept` when only available.